eng/common/scripts/Package-Properties.ps1

```diff
@@ -13,6 +13,7 @@ class PackageProps {
     [string]$SdkType
     [boolean]$IsNewSdk
     [string]$ArtifactName
+    [string]$ModuleName
```

Comment:
If we can avoid it, I would prefer we not add a new property. I know other languages use ArtifactName for different things and it could potentially be used for module name as well.

Reply:
I can remove this as it turned out I don't even need it here. Go is not setting ArtifactName but instead Module name is added as `NotePropertyName` but it does not show up whren the properties are written to the `PackageInfo` file. Anyways the name property is sufficient for what I need to accomplish right now.